BatchApex/BatchLeadSchedule.cls

```diff
@@ -0,0 +1,9 @@
+/**
+ * Schedulable class for to delete the duplicate leads
+ */
+global with sharing class BatchLeadSchedule implements Schedulable {
```

Comment:
we give the cron expr. at the time of excution
in the System.Schedule('BatchName',' 0 0 0/3 * * ?', classInstance);